src/app/content/utils/archiveTreeUtils.spec.ts

```diff
@@ -174,3 +175,25 @@ describe('nodeHasId', () => {
     expect(nodeHasId('some-id', node)).toEqual(false);
   });
 });
+
+describe('comparePositionsOfNodes', () => {
+  it('return 0 for the same ids', () => {
+    expect(comparePositionsOfNodes(book.tree, '1', '1')).toEqual(0);
+  });
+
+  it('return 0 for ids that does not belongs to this book', () => {
+    expect(comparePositionsOfNodes(book.tree, '1', '2')).toEqual(0);
+  });
```

Comment:
I think this is more consistent with `if (idA === idB) { return 0; }`, because provided ids may also not belong to this book, but I don't have a strong opinion on this so I've added an error at the end of the function